x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/components/rules_table_columns/status.tsx

```diff
@@ -6,15 +6,55 @@
  */
 
 import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiHorizontalRule, EuiText } from '@elastic/eui';
+import { RuleTranslationResult } from '../../../../../common/siem_migrations/constants';
 import type { RuleMigration } from '../../../../../common/siem_migrations/model/rule_migration.gen';
 import * as i18n from './translations';
 import type { TableColumn } from './constants';
 import { StatusBadge } from '../status_badge';
+import { TableHeader } from './header';
+import { convertTranslationResultIntoText } from '../../utils/translation_results';
 
 export const createStatusColumn = (): TableColumn => {
   return {
     field: 'translation_result',
-    name: i18n.COLUMN_STATUS,
+    name: (
+      <TableHeader
+        title={i18n.COLUMN_STATUS}
+        tooltipContent={
+          <FormattedMessage
+            id="xpack.securitySolution.siemMigrations.rules.tableColumn.statusTooltip"
+            defaultMessage="{title}
+            {installed} - already added to Elastic SIEM. Click View to manage and enable it. {lineBreak}
+            {translated} - rule is ready to install. Rules with matching capabilities have been mapped to Elastic Authored rules. If not match was detected, an AI translation was provided. {lineBreak}
+            {partiallyTranslated} - part of the original query could not be translated. Make sure you’ve uploaded all macros and lookups, and resolved all syntax errors. {lineBreak}
+            {notTranslated} - none of the original query could be translated."
+            values={{
+              lineBreak: <br />,
+              title: (
+                <>
```

Comment:
I see this `<>` fragment inside all the `title` entries, is it necessary?

Reply:
Ahh good catch! I had a different structure here initially and then forgot to remove obsolete `<>` once I updated components.

Reply:
done in https://github.com/elastic/kibana/pull/207177/commits/b2eb7a39331f5a61aa2d4cff92012b2b8452e3ea